x-pack/plugins/alerting/common/alert_schema/index.ts

```diff
@@ -6,4 +6,6 @@
  */
 
 export { alertFieldMap } from './field_maps/alert_field_map';
+export { AlertSchema } from './schemas/alert_schema';
+export type { Alert } from './schemas/alert_schema';
```

Comment:
The `Alert` type is auto-generated from the index mapping and reflects the fields that the framework will populate in the alerts as data documents.